plugins/modules/cloud/misc/proxmox.py

```diff
@@ -140,9 +140,10 @@
     type: str
   unprivileged:
     description:
-      - Indicate if the container should be unprivileged
+      - Indicate if the container should be unprivileged.
+      - The default value for this param is C(false) but that is being deprecated.
+      - It will be replaced with C(true) in community.general 7.0.0
```

Comment:
@russoz Maybe worth to expand a bit on the deprecation reason, and warn not to use the default both here and in console? 
Unless when there's a security risk people tend to ignore deprecation messages for longer IMO. wdyt?

Reply:
The `module.deprecate()` part in the code will print a message out in the console when running the playbook, so I guess that is covered.

That being said, in version 7.0.0 the old default value will be removed and no longer mentioned. If you want to expand a bit on the deprecation reason, I reckon the documentation block is the right place to do that. Just keep in mind that this will be published in the official documentation at:

https://docs.ansible.com/ansible/latest/collections/community/general/proxmox_module.html

Long documentations tend to be ignored, so there's a balance to strike. Perhaps if you refer to some page that explains why having privileged VMs by default is a bad idea (yeah, it might be not obvious to some people), that might help.